### PR TITLE
#162758871 Add test for updating lists

### PR DESCRIPTION
**Pull request by kwameasiago**
#### What does this PR do?
This PR defines test for updating list items

#### Description of Task to be completed?
Have tests for updating items

#### How should this be manually tested?

- Clone the project locally

- cd into the `Bucketlist` folder

- Run the tests. They should fail

### What are the relevant pivotal tracker stories
[#162758871](https://www.pivotaltracker.com/story/show/162758871)

### CODE REVIEW

File: djangorestapi/api/tests.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 from django.test import TestCase
 from .models import ToDoList
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.core.urlresolvers import reverse
+
 
 class ModleTestList(TestCase):
```
Comment (KMaina) [inline djangorestapi/api/tests.py]: Check the spelling of 'Model'

File: djangorestapi/api/tests.py [hunk 2]
```diff
@@ -17,3 +21,35 @@ def test_model_can_create_a_todo_list(self):
         self.todo_list.save()
         new_count = ToDoList.objects.count()
         self.assertNotEqual(initial_count, new_count)
+
```
Comment (KMaina) [inline djangorestapi/api/tests.py]: Remove this line of white space

File: djangorestapi/api/tests.py [hunk 3]
```diff
@@ -17,3 +21,35 @@ def test_model_can_create_a_todo_list(self):
...
+
```
Comment (KMaina) [inline djangorestapi/api/tests.py:26]: Remove this line of white space

**Comment by KMaina** [comment]
Good job covering a few edge cases in your tests.

File: djangorestapi/api/tests.py [hunk 4]
```diff
@@ -17,3 +21,33 @@ def test_model_can_create_a_todo_list(self):
...
+class TestUpdateList(TestCase):
+    """Test for updating list items"""
+    
+    # define variable needed to run test
+    def setUp(self):
+        self.client = APIClient()
+        self.update_data = {'item_name': 'change item'}
+        self.res = self.client.post(reverse('create'),self.update_data,format='json')
```
Comment (Georgeygigz) [inline djangorestapi/api/tests.py]: Good work so far, but I would recommend that you use descriptive variable names. i.e instead of using (res) you can use response

**Comment by KMaina** [comment]
Moving forward try and make sure that the description for your PR is a bit more descriptive. You can follow the guide from -> https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#pr-description-template-markdown
